lambda/deletePass/index.js

```diff
@@ -75,7 +75,7 @@ exports.handler = async (event, context) => {
 
       return sendResponse(200, { msg: 'Cancelled' }, context);
     } else if (event.queryStringParameters.passId && event.queryStringParameters.park) {
-      if ((await checkPermissions(event)) === false) {
+      if ((await checkPermissions(event)).decoded === false) {
```

Comment:
Nit: slightly safer to check for `!== true` in case typo in property name or something results in `undefined`

Reply:
Fixed in b07f491.